src/index.html

```diff
@@ -11,17 +11,410 @@
       rel="stylesheet"
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/index.css"
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__img"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo header logo"
+        />
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            data-qa="nav-hover"
+            class="header__item"
+          >
+            <a
+              href="#"
+              class="header__link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="cards">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__image">
+          <img
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+          />
+        </div>
+
+        <div class="card__content">
+          <div class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__reviews">
+            <div class="stars card__stars">
+              <div class="stars__star stars__star--color"></div>
+              <div class="stars__star stars__star--color"></div>
+              <div class="stars__star stars__star--color"></div>
+              <div class="stars__star stars__star--color"></div>
+              <div class="stars__star"></div>
```

Comment:
The way you implemented stars is not very flexible. Consider making it a separate component with a parameter for the number of active stars. This way you can easily change the number of active stars for each product.